pkg/conversion/mapi2capi/aws_fuzz_test.go

```diff
@@ -145,6 +145,7 @@ func awsProviderSpecFuzzerFuncs(codecs runtimeserializer.CodecFactory) []interfa
 			c.FuzzNoCustom(ebs)
 
 			// Fuzz required fields so that they are not empty.
+			// Setting volumeSize to a random int64 value.
```

Comment:
It seems the fuzz test is already setting the volumeSize if it's empty (to a random int64 value). Is having it constantly set at 120GiB what we want here, or is there any value in having it set to a random value?